src/cgame/cg_draw_hud.c

```diff
@@ -57,7 +57,7 @@ const hudComponentFields_t hudComponentFields[] =
 	{ HUDF(xptext),             CG_DrawXP,                        0.25f,  { "Draw Suffix" } },
 	{ HUDF(ranktext),           CG_DrawRank,                      0.20f,  { 0 } },
 	{ HUDF(statsdisplay),       CG_DrawSkills,                    0.25f,  { "Column" } },
-	{ HUDF(weaponicon),         CG_DrawGunIcon,                   0.19f,  { "Icon Flash",    "Dynamic Heat Color" } },
+	{ HUDF(weaponicon),         CG_DrawGunIcon,                   0.19f,  { "Icon Flash",    "Dynamic Heat Color", 		   "Only Charge" } },
```

Comment:
Only charge doesn't sound correct as it could be confused with other weapon using charge, which seems confusing. 
May I suggest separate in 2 differents options "Only ticking" and "Show overheat" indicator ? or another wording.

Moreover, it could a nice plus to separate overheat indicator in his own comp, so it could be move aeound separatly from weapon icon (or stack up like currently)

Another option